src/translations/en.json

```diff
@@ -234,5 +234,6 @@
   "claim.ClaimPercentageReferralsReport.region": "Region",
   "claim.ClaimPercentageReferralsReport.district": "District",
   "claim.ClaimPercentageReferralsReport.dateStart": "Date From",
-  "claim.ClaimPercentageReferralsReport.dateEnd": "Date To"
+  "claim.ClaimPercentageReferralsReport.dateEnd": "Date To",
+  "claim.referHF": "Referal HF"
```

Comment:
Try to extend a little bit this key. Usually, we're using structure like: _"module.component.object"_. But this is not mandatory, just a good pratice.